### PR TITLE
feature: add array property editor

### PR DESCRIPTION
**Pull request by bzp2010**
add feature #166 
Make array properties available for graphical editing
Add translation content
image

### CODE REVIEW

File: src/components/PluginDialog/index.vue [hunk 1]
```diff
@@ -132,6 +132,26 @@
             active-color="#13ce66"
             inactive-color="#ff4949"
           />
+
+          <!-- array property -->
+          <div v-if="schema.properties[key].type === 'array'">
+            <!-- @input="onPropertyChange(key, $event)" -->
```
Comment (juzhiyuan) [inline src/components/PluginDialog/index.vue:138]: Please remove this line.

File: src/components/PluginDialog/index.vue [hunk 2]
```diff
@@ -132,6 +132,26 @@
...
+            <el-input
+              v-for="(arrayIndex) in arrayPropertiesLength[key]"
+              :key="arrayIndex"
+              v-model="data[key][arrayIndex]"
+              :placeholder="key + ' [' + (arrayIndex) + ']'"
+              style="margin-top:5px;"
```
Comment (juzhiyuan) [inline src/components/PluginDialog/index.vue:144]: use style selector instead.

File: src/components/PluginDialog/index.vue [hunk 3]
```diff
@@ -132,6 +132,26 @@
...
           />
+
+          <!-- array property -->
+          <div v-if="schema.properties[key].type === 'array'">
+            <!-- @input="onPropertyChange(key, $event)" -->
+            <el-input
+              v-for="(arrayIndex) in arrayPropertiesLength[key]"
+              :key="arrayIndex"
+              v-model="data[key][arrayIndex]"
+              :placeholder="key + ' [' + (arrayIndex) + ']'"
```
Comment (juzhiyuan) [inline src/components/PluginDialog/index.vue:143]: Please use ` `${}` ` instead

File: src/lang/en.ts [hunk 4]
```diff
@@ -85,5 +85,8 @@ export default {
     },
     keyTip: 'You can edit and input any value here,and press Enter to confirm.',
     hashOnTip: 'Select a hash on.'
+  },
+  message: {
+    cannotAddMore: 'You cannot add more item!'
```
Comment (juzhiyuan) [inline src/lang/en.ts:90]: `more items`

File: src/components/PluginDialog/index.vue [hunk 5]
```diff
@@ -242,8 +263,37 @@ export default class extends Vue {
...
+        schemaKeys[key] = schema.properties[key].default
+        continue
+      }
+
+      switch (schema.properties[key].type) {
+        case 'array':
+          schemaKeys[key] = []
+          this.arrayPropertiesLength[key] = [...new Array(this.pluginData[key] ? this.pluginData[key].length : schema.properties[key].minItems).keys()]
+          break
+
```
Comment (juzhiyuan) [inline src/components/PluginDialog/index.vue:279]: codes clean.

File: src/components/PluginDialog/index.vue (hunk 1 above)
Comment (bzp2010) [reply src/components/PluginDialog/index.vue]: fixed😀

File: src/components/PluginDialog/index.vue (hunk 3 above)
Comment (bzp2010) [reply src/components/PluginDialog/index.vue]: fixed😀

File: src/components/PluginDialog/index.vue (hunk 2 above)
Comment (bzp2010) [reply src/components/PluginDialog/index.vue]: fixed😀

File: src/lang/en.ts (hunk 4 above)
Comment (bzp2010) [reply src/lang/en.ts]: I made some changes, but I don't know if that's what you meant.

File: src/components/PluginDialog/index.vue (hunk 5 above)
Comment (bzp2010) [reply src/components/PluginDialog/index.vue:283]: I don't understand. Can you tell me in more detail? 😁

File: src/lang/en.ts (hunk 4 above)
Comment (juzhiyuan) [reply src/lang/en.ts]: We would use `items` here, because `more` means many items :)

File: src/components/PluginDialog/index.vue (hunk 5 above)
Comment (juzhiyuan) [reply src/components/PluginDialog/index.vue:283]: Please remove the blank line in `switch` statement :)

File: src/lang/en.ts (hunk 4 above)
Comment (bzp2010) [reply src/lang/en.ts]: such as `cannotAddMoreItems` ?

**Pull request merged**